index.html

```diff
@@ -29,22 +30,205 @@ <h1>
+          <ul class="links">
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">HTML</a></li>
+        </ul>
+        <div class="button"><a href="#">See project</a></div>
+      </div>
+      </div>
+<!--3-->
+      <div class="item">
+        <div class="border">
+          <div class="border-image">
+          <img src="images/grid.svg" alt="">
+        </div>
+        </div>
+        <div class="content">
+          <div class="title"><p>
+            Multi-Post Stories <br>Gain+Glory
+            </p></div> 
+        <ul class="links">
+          <li><a href="#">Ruby on Rails</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+          <li><a href="#">HTML</a></li>
+      </ul>   
+      <div class="button"><a href="#">See project</a></div>
+    </div>
+
+<!--4-->
+    <div class="item">
+      <div class="border">
+        <div class="border-image">
+        <img src="images/grid.svg" alt="">
+      </div>
+      </div>
+      <div class="content">
+        <div class="title"><p>
+          Multi-Post Stories <br>Gain+Glory
+          </p></div>
+      <ul class="links">
+        <li><a href="#">Ruby on Rails</a></li>
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">JavaScript</a></li>
+        <li><a href="#">HTML</a></li>
+    </ul>
+    <div class="button"><a href="#">See project</a></div>
+  </div>
+
+<!--5-->
+  <div class="item">
+    <div class="border">
+      <div class="border-image">
+      <img src="images/grid.svg" alt="">
+    </div>
+    </div>
+    <div class="content">
+      <div class="title"><p>
+        Multi-Post Stories <br>Gain+Glory
+        </p></div>
+    <ul class="links">
+      <li><a href="#">Ruby on Rails</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">JavaScript</a></li>
+      <li><a href="#">HTML</a></li>
+  </ul>
+  <div class="button"><a href="#">See project</a></div>
+</div>
+
+<!--6-->
+<div class="item">
+  <div class="border">
+    <div class="border-image">
+    <img src="images/grid.svg" alt="">
+  </div>
+  </div>
+  <div class="content">
+    <div class="title"><p>
+      Multi-Post Stories <br>Gain+Glory
+      </p></div>
+  <ul class="links">
+    <li><a href="#">Ruby on Rails</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+    <li><a href="#">HTML</a></li>
+</ul>
+<div class="button"><a href="#">See project</a></div>
+</div>
+</div>
+</section>
+
+<!--About-->
+<section class="about">
+  <div class="about-me-text">
+      <h2 class="about-text">About me</h2>
+      <div class="about-heading">I can help you build a product , <br>feature or website 
+        Look through <br>some of my work and
+         experience! If <br>you like what
+          you see and have a <br>project you
+           need coded,<br> don’t hesitate to contact me.</div>
```

Comment:
- [ ] As I suggested in the above comment, and also stated in the project requirement on following [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md),  you need to use the correct tags over divs
> Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.
- - I'll suggest you put the text content in a `p tag`. 😄 😄